qa/rpc-tests/mergetoaddress_sapling.py

```diff
@@ -16,7 +16,6 @@ def setup_chain(self):
     def setup_network(self, split=False):
         self.helper.setup_network(self, [
             '-anchorconfirmations=1',
-            '-allowdeprecated=legacy_privacy',
```

Comment:
:tada:

Reply:
As I read this, we are no longer testing `LegacyPrivacy`. Maybe we should.

Reply:
We test legacy_privacy in a number of other places, and it’s all handled internally in WalletTxBuilder, so the logic is shared between operations … but I do think we should have more tests directly on `WalletTxBuilder` than we do.